src/core/src/core_logic/ConfigurePatchingProcessor.py

```diff
@@ -73,7 +73,7 @@ def __try_set_patch_mode(self):
 
             self.current_auto_os_patch_state = self.package_manager.get_current_auto_os_patch_state()
 
-            if self.current_auto_os_patch_state == Constants.AutomaticOSPatchStates.UNKNOWN:
+            if self.execution_config.patch_mode == Constants.PatchModes.AUTOMATIC_BY_PLATFORM and self.current_auto_os_patch_state == Constants.AutomaticOSPatchStates.UNKNOWN:
                 # NOTE: only sending details in error objects for customer visibility on why patch state is unknown, overall configurepatching status will remain successful
                 self.__report_consolidated_configure_patch_status(error="Extension attempted but could not disable some of the auto OS update service. Please check if the auto OS services are configured correctly")
```

Comment:
nit: "could not disable some of the auto OS update service"  --> one or more automatic OS update services
Not necessary for this PR; calling out.

Reply:
Resolved